### ISSUE TITLE
Fatal Exception: java.io.IOException: Fetch failed: Read timed out

### ISSUE THREAD

**Issue by nizam-betterapp** [Abhimanyu14/compose-emoji-picker#1]
Thanks for the great library. Can you please help me fix this crash that happens for few users? I want the app to not crash but show some error message a tleast.

```
Fatal Exception: java.io.IOException: Fetch failed: Read timed out
       at emoji.core.datasource.EmojiDataSourceImpl$getAllEmojis$2$1.onFetchFailure(EmojiDataSourceImpl.java:38)
       at emoji.core.emojifetcher.EmojiFetcherImpl$fetchEmojiData$1.onFailure(EmojiFetcherImpl.kt:45)
       at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:525)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
       at java.lang.Thread.run(Thread.java:1012)
```

**Comment by Abhimanyu14** [comment]
Hi,
Issue fixed in the latest version - 1.0.0-alpha11.

**Comment by nizam-betterapp** [comment]
Thanks @Abhimanyu14 . I could see this crash not being reported now.
